controls/anssi.yml

```diff
@@ -694,13 +694,17 @@ controls:
     level: high
     title: Setting SELinux booleans
     description: >-
-      It is recommended to set the following Boolean values to off:
-      allow_execheap        if off, forbid processes to make their heap executable (heap);
-      allow_execmem         if off, forbid processes to have a memory area with rights w (write) and x (execute);
-      allow_execstack       if off, forbid processes to make their stack (stack) executable;
-      secure_mode_insmod    if off, prohibits dynamic loading of modules by any process;
-      ssh_sysadm_login      if off, forbid SSH logins to connect directly in sysadmin role.
+      It is recommended to set the following Booleans:
+      allow_execheap to off, forbids processes to make their heap executable;
+      allow_execmem to off, forbids processes to have both write and execute rights on memory pages;
+      allow_execstack to off, forbids processes to make their stack executable;
+      secure_mode_insmod to on, prohibits dynamic loading of modules by any process;
+      ssh_sysadm_login to off, forbids SSH logins to connect directly in sysadmin role.
     rules:
+    - sebool_selinuxuser_execheap
```

Comment:
Is it a different boolean than allow_execheap or has it been renamed?

Do we have rules for other booleans?

Reply:
It is a different boolean, I'll update this.